doc/diffview.txt

```diff
@@ -213,6 +217,11 @@ win_config                                      *diffview-config-win_config*
                 current window.
                 Default: `0`
 
+            {win_opts} (table)
+                Any vim options to set for the local window (see |vim.opt_local|).
+                You don't have to copy all the defaults, just specify any
+                values you want to set/override.
```

Comment:
> You don't have to copy all the defaults ...

This doesn't really make sense now that the defaults are empty. Maybe change this paragraph to something like:

```vimhelp
Table of window local options (see |vim.opt_local|). These options are applied
whenever the window is opened.
```